schemas/context/opportunity-contactrole.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/opportunity-contactrole",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Opportunity Contact Role mixin",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/opportunitycontactrole"
+  ],
+  "description": "Opportunity Contact Role additional properties",
+  "definitions": {
+    "opportunityContactRole": {
+      "properties": {
+        "xdm:opportunityContactRole": {
```

Comment:
Schema $id and the definition name are generally same. Should we make one similar to the other ?